### DISCUSSION TITLE
Do I need to use Visual Studio C++ to be able to test what I do in HaxeFlixel?

### DISCUSSION THREAD

**Discussion by QuetzalcoutlDev** [HaxeFlixel/flixel#2827] [Q&A]
I was looking to start with HaxeFlixel to be able to develop games but I have a question that I saw when trying to test the flixel samples

Do I have to have Visual Studio Community 2022 and the C++ tools installed just to be able to test simple projects?

Because the truth download 12 GB or more is too much and it makes it difficult for me to download it.

**Comment by Snirozu** [comment]
https://lime.openfl.org/docs/getting-started/targets/

**Reply by QuetzalcoutlDev** [reply]
I read again, and if I need to have Visual Studio C++ installed

Ok I'll have to download it

**Reply by Geokureli** [reply]
you shouldn't* need to install  all the features of Visual Studio C++, I believe mine was only a couple GB
